frontend/src/app/Settings/DeviceSettings.jsx

```diff
@@ -42,6 +53,10 @@ const DeviceSettings = ({ deviceSettings, updateDeviceSettings }) => {
     );
   }
 
+  if (error) {
```

Comment:
Is React smart enough to rethrow an error that is `return`ed from a render function?